src/ape_ethereum/abi.py

```diff
@@ -0,0 +1,15 @@
+from typing import Any, Dict, List, Tuple, Union
+from ape.utils import is_array
+
+def parse_type(type: Dict[str, Any]) -> Union[str, Tuple, List]:
```

Comment:
I would put this in `ecosystem.py` somewhere, maybe as `_parse_type()` or inline the functionality into the spot where it's used, no need to export it.. the goal is make it not public anymore

Reply:
One issue is that it's still used in tracing, `ape/utils/trace.py` in the `decode_calldata()` method.